coderd/provisionerjobs.go

```diff
@@ -287,9 +287,10 @@ func convertProvisionerJobLog(provisionerJobLog database.ProvisionerJobLog) code
 
 func convertProvisionerJob(provisionerJob database.ProvisionerJob) codersdk.ProvisionerJob {
 	job := codersdk.ProvisionerJob{
-		ID:        provisionerJob.ID,
-		CreatedAt: provisionerJob.CreatedAt,
-		Error:     provisionerJob.Error.String,
+		ID:         provisionerJob.ID,
+		CreatedAt:  provisionerJob.CreatedAt,
+		Error:      provisionerJob.Error.String,
+		SourceHash: provisionerJob.StorageSource,
```

Comment:
I think we keep this as `StorageSource` for rn for API <-> DB naming consistency